packages/vaadin-grid/src/vaadin-grid-helper.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+
+/**
+ * @param {Array<Object>} columns array of columns to be modified
+ * @param {number} scope multiplier added to base order for each column
+ * @param {number} baseOrder base number used for order
+ */
+export function updateColumnOrders(columns, scope, baseOrder) {
+  let c = 1;
+  columns.forEach((column, _) => {
```

Comment:
nit: `_` is not being used, so its declaration can be removed.

nit2: if this file is supposed to hold future helpers, I believe its name could be changed to `vaadin-grid-helpers.js` instead.